mlrun/projects/project.py

```diff
@@ -2040,6 +2045,63 @@ def setup(self, save: bool = True) -> "MlrunProject":
         )
         return _run_project_setup(self, setup_file_path, save)
 
+    def create_model_monitoring_alert_configs(
+        self,
+        name: str,
+        summary: str,
+        endpoints: list[mlrun.model_monitoring.model_endpoint.ModelEndpoint],
+        events: Union[list[alert_constants.EventKind], alert_constants.EventKind],
+        notifications: list[alert_constants.AlertNotification],
+        result_names: Optional[
+            list[str]
+        ] = None,  # can use wildcards - see below for explanation.
+        severity: alert_constants.AlertSeverity = alert_constants.AlertSeverity.MEDIUM,
+        criteria: alert_constants.AlertCriteria = alert_constants.AlertCriteria(
+            count=1, period="10m"
+        ),
+        reset_policy: mlrun.common.schemas.alert.ResetPolicy = mlrun.common.schemas.alert.ResetPolicy.AUTO,
+    ) -> list[mlrun.alerts.alert.AlertConfig]:
+        db = mlrun.db.get_run_db(secrets=self._secrets)
+        metrics = []
+        alerts = []
+        # TODO: Refactor to use a single request to improve performance at scale, ML-8473
+        for endpoint in endpoints:
+            metrics_by_endpoint = db.get_model_endpoint_monitoring_metrics(
+                project=self.name, endpoint_id=endpoint.metadata.uid
+            )
+            metrics_fqn_by_endpoint = [
+                get_result_instance_fqn(
```

Comment:
I don't think you need this, the full name is already part of metric/result.

Reply:
full name and fqn is not identical:

**full name** starting with `project.name` - for example "my_project.my_app.result.my_result" - as described in `ModelEndpointMonitoringMetric._compose_full_name`

**fqn**  starting with `model_endpoint.metadata.uid` for example "my_endpoint.my_app.result.my_result" - as described in `get_result_instance_fqn`